### PR TITLE
Kubestatus

### PR DESCRIPTION
**Pull request by jkurambhatti**
Add a new button `Kube Status` in the deploy environment page.
This will show all the information (Pods, RC, SVC, ING) related to the app.

### CODE REVIEW

File: app/assets/javascripts/pages/kube_status.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import superagent from "superagent";
+import _ from "lodash";
+
+class KubeStatusPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      appStatus: this.props.appStatus
+    };
+    this.render = require("./kube_status.rt");
```
Comment (himani39) [inline app/assets/javascripts/pages/kube_status.js]: Why use `rt`? Why not write the render method using `jsx`?

Comment (jkurambhatti) [reply app/assets/javascripts/pages/kube_status.js]: Just followed the existing conventions

File: app/assets/javascripts/pages/kube_status.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import ReactDom from "react-dom";
+import superagent from "superagent";
```
[the remark this answers is not in the document]
Comment (jkurambhatti) [reply app/assets/javascripts/pages/kube_status.js]: will remove this

File: app/assets/javascripts/pages/kube_status.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import _ from "lodash";
```
[the remark this answers is not in the document]
Comment (jkurambhatti) [reply app/assets/javascripts/pages/kube_status.js]: will remove this

**Pull request merged**